t/sql/selenium.sql

```diff
@@ -55,9 +55,14 @@ INSERT INTO event (id, gid, name, begin_date_year, begin_date_month, begin_date_
 INSERT INTO event_gid_redirect (gid, new_id, created) VALUES
     ('a428f34a-9761-3544-2279-a58cbf15ba9d', 1606, '2014-12-18 13:00:34.106501+00');
 
+DROP TRIGGER a_ins_instrument ON instrument;
+
 INSERT INTO instrument (id, gid, name, type, edits_pending, last_updated, comment, description) VALUES
     (55, '7ee8ebf5-3aed-4fc8-8004-49f4a8c45a87', 'electric guitar', 2, 0, '2014-08-22 17:09:51.69681+00', '', '');
 
+CREATE TRIGGER a_ins_instrument AFTER INSERT ON instrument
+    FOR EACH ROW EXECUTE PROCEDURE a_ins_instrument();
+
```

Comment:
I must've added it at some point to prevent it from adding instruments to the link_attribute_type table, but whatever I needed that for it doesn't seem to be necessary anymore, so removed it